src/sage/schemes/elliptic_curves/ell_finite_field.py

```diff
@@ -1341,6 +1343,96 @@ def is_ordinary(self, proof=True):
         """
         return not is_j_supersingular(self.j_invariant(), proof=proof)
 
+    def has_order(self, value, num_checks=8):
```

Comment:
Add an optional `proof` argument which should default to `True`.

Reply:
That's out of scope for this PR, the method is probabilistic as noted in `.. NOTE:` already.